src/integrationTest/java/uk/gov/moj/sdt/controllers/FeedbackControllerIT.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.moj.sdt.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.moj.sdt.base.WireMockBaseTest;
+import uk.gov.moj.sdt.request.CMCFeedback;
+
+import java.util.Calendar;
+import javax.inject.Inject;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class FeedbackControllerIT extends WireMockBaseTest {
+
+    private String CLAIM_ERROR = "Claim Error";
+
+    private String CLAIM_NUMBER = "123456";
+
+    private String ERROR_CODE = "1234";
+
+    private static final String SDT_REFERENCE = "SDT_REQ_TEST_1";
+
+    private static final String POST_CMC_URL = "/CMCFeedback";
+
+    @Inject
+    private WebApplicationContext wac;
+
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    public void setUp() {
+        super.initMock();
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+    }
+
+    @Test
+    @Sql(scripts = {"classpath:sql/IndividualRequestDaoTest.sql"})
+    public void shouldCreateClaim() throws Exception {
```

Comment:
Can the public modifier also be removed from the test methods?

Reply:
removed public modifier